evalml/tests/dependency_update_check/minimum_test_requirements.txt

```diff
@@ -1,5 +1,5 @@
 IPython==8.10.0
-PyYAML==5.4
+PyYAML==6.0.1
```

Comment:
Honestly don't remember if this was needed so need to test to see if I can lower it back down.